pkg/apis/admission/types.go

```diff
@@ -73,6 +73,10 @@ type AdmissionRequest struct {
 	// OldObject is the existing object. Only populated for UPDATE requests.
 	// +optional
 	OldObject runtime.Object
+	// DryRun indicates that modifications will definitely not be persisted for this request.
```

Comment:
And that the call must not have side effects.

Reply:
+1, what about something also indicating that a value of DryRun false doesn't guarantee that the object will be persisted, so any side effects should have a reconciliation mechanism, or is that redundant because it's already somewhere in the webhook documentation?

Reply:
It should be documented, but feel free to state it again here if you think it makes it clearer.

Reply:
Okay, added this